run/local-cluster.sh

```diff
@@ -139,6 +139,11 @@ fi
 # Fetch the local status contents from node 1 and verify build information is present.
 LOCAL_URL="$DOCKERHOST:${HTTP_PORTS[1]}/_status/local/"
 LOCAL=$(curl --noproxy '*' -s $LOCAL_URL)
+if [[ -z $LOCAL ]]; then
```

Comment:
Why did you add this? Is it possible, btw, that the container hasn't yet started up and this fails? That wouldn't quite be a condition which warrants failure.